tools/mcsema_disass/ida/get_cfg.py

```diff
@@ -1040,11 +1040,21 @@ def recover_regions(M, exported_vars, global_vars=[]):
     seg_name = idc.SegName(seg_ea)
     seg_names[seg_ea] = seg_name
 
-    if not is_external_segment_by_flags(seg_ea) or \
-       segment_contains_external_function_pointers(seg_ea):
+    if (not is_external_segment_by_flags(seg_ea) or \
+       segment_contains_external_function_pointers(seg_ea)) and \
```

Comment:
indent by one space, so that the `segment` is in line with the `not`, and not in line with the `(`.

Reply:
Can you add a comment that explicitly references `Issue #330`, so that people can understand the decision. Perhaps have a bit of verbose comment here that says that functions like `__libc_csu_init` will often iterate over `.init_array` by using a start/end variable to bound the iteration, but the `end` variable can be in the `.fini_array` section, so we force them to be lifted as a single LLVM global variable.